.gitlab-ci.yml

```diff
@@ -387,6 +387,7 @@ pup6.pe-ldap:
   <<: *acceptance_base
   script:
     - 'bundle exec rake beaker:suites[ldap,default]'
+    - 'bundle exec rake beaker:suites[ds389,default]'
```

Comment:
Please break this out into its own job so it can be run concurrently.  This will speed up the execution of the tests and allow independent reporting of results.